cuizinart/monitor.py

```diff
@@ -0,0 +1,99 @@
+import time
+import os
+from datetime import datetime
+from metadata_schema import *
+from watchdog.observers import Observer
+from watchdog.events import PatternMatchingEventHandler
+from extract_netcdf_header import *
+
+
+def parse_str_time(strs):
+    return datetime.strptime(strs, '%Y-%m-%d %H:%M:%S')
+
+
+def add_metadata(src_path):
+    product_name = src_path.split('/')[-2]
+    file_name = src_path.split('/')[-1]
+    product = Product.query.filter_by(key=product_name).first()
+    if product is None:
+        print("Only support existing products")
+    else:
+        try:
+            f = Dataset(src_path,'r')
+        except Exception as e:
+            LOGGER.exception('Missing File {}'.format(src_path))
```

Comment:
Where is `LOGGER` coming from? Is this the one defined in extract-netcdf-header.py? If so, I don't think we should use it here. Given that we `print()` everywhere else so far, we could do the same here until we implement proper logging.

Reply:
Yes, we can change to print on all.